File: filter-api/src/models/vuelo.ts
import { Entity, Column, PrimaryGeneratedColumn } from "typeorm";

@Entity()
export class Vuelo {

    constructor(
        origen?: string,
        destino?: string,
        fecha_salida?: string,
        fecha_llegada?: string,
        personas?: number,
        precio?: number,
        aerolinea?: string,
        enlace?: string
    ) {
        this.origen = origen;
        this.destino = destino;
        this.fecha_salida = fecha_salida;
        this.fecha_llegada = fecha_llegada;
        this.personas = personas;
        this.precio = precio;
        this.aerolinea = aerolinea;
        this.enlace = enlace;
    }

    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    origen: string;

    @Column()
    destino: string;

    @Column()
    fecha_salida: string;

    @Column()
    fecha_llegada: string;

    @Column()
    personas: number;

    @Column()
    precio: number;

    @Column()
    aerolinea: string;

    @Column()
    enlace: string;

}

File: filter-api/api/models/vuelo.py

class Vuelo():

    def __init__(self, origen, destino, fecha_salida, fecha_llegada, personas, precio, aerolinea, enlace):
        self.origen = origen
        self.destino = destino
        self.fecha_salida = fecha_salida
        self.fecha_llegada = fecha_llegada
        self.personas = personas
        self.precio = precio
        self.aerolinea = aerolinea
        self.enlace = enlace

    def to_json(self):
        return {
            "origen": self.origen,
            "destino": self.destino,
            "fecha_salida": self.fecha_salida,
            "fecha_llegada": self.fecha_llegada,
            "personas": self.personas,
            "precio": self.precio,
            "aerolinea": self.aerolinea,
            "enlace": self.enlace,
        }

    def __repr__(self):
        return '<Vuelo {} - {}>'.format(self.origen, self.destino)